credential_helper.py

```diff
@@ -0,0 +1,32 @@
+import configparser
+import json
+import requests
+
+
+class CredentialHelper:
+    def __init__(self, config_file="config.ini"):
+        self.config = configparser.ConfigParser()
+        self.config.read(config_file)
+
+    def get_access_token(self):
+        if "credentials" not in self.config:
+            raise "config file does not include credentials"
+
+        if "client_id" not in self.config["credentials"] or "client_secret" not in self.config["credentials"]:
+            raise "credentials section does not contain client_id or client_secret"
+
+        client_id = self.config["credentials"]["client_id"]
+        client_secret = self.config["credentials"]["client_secret"]
+
+        params = {
+            "client_id": client_id,
+            "client_secret": client_secret,
+            "grant_type": "client_credentials",
+        }
+        res = requests.post(
+            "https://api.oregonstate.edu/oauth2/token", params)
+        res.raise_for_status()
+
+        data = json.loads(res.text)
+
+        return data["access_token"]
```

Comment:
My thought was no, since, and correct me if I'm wrong, the returned json is guaranteed to contain `access_token` if no error occurs, and we've already checked for errors in the `res.raise_for_status` statement

Reply:
yes and no, lol. We've tried our best to make sure our APIs return correct data that follows JSON API schema. That from API development team perspective, however, when you're writing the integration test script (like what you're doing right now), you shouldn't assume the data you got is always correct. You should make sure your script can catch all of the errors and handle them nicely.

Reply:
Ah, that makes sense. I'll add a check for it 👍